cmake/BLTPrivateMacros.cmake

```diff
@@ -8,6 +8,52 @@ include(CMakeParseArguments)
 ## Internal BLT CMake Macros
 
 
+##-----------------------------------------------------------------------------
+## blt_determine_scope(TARGET <target>
+##                     SCOPE  <public (Default)| interface | private >
```

Comment:
It might be worthwhile adding a note here that the default is ``PUBLIC`` for regular targets, but ``INTERFACE`` for ``INTERFACE_LIBRARY`` targets. This might help avoid user confusion.